lib/controls/sync_done_control.js

```diff
@@ -0,0 +1,88 @@
+// Copyright 2011 Mark Cavage, Inc.  All rights reserved.
+
```

Comment:
These new files should not start with this outdated copyright notice. They should also start with `'use strict'`.
```suggestion
'use strict'

```